.github/workflows/tag-release-drafter.yml

```
@@ -0,0 +1,34 @@
+name: Tag Release drafter
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Initialize git config for PR author # https://github.com/actions/checkout/issues/13
+      run: |
+       git config user.name "github-actions[bot]"
+       git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+    - name: Get package.json version
+      id: package
+      uses: uwblueprint/sdc-actions/actions/get-package-version@main
+
+    - name: Create tag
+      run: git tag -a v${{steps.package.outputs.version}} -m "Automated tag v${{steps.package.outputs.version}} pushed by ${{github.actor}}"
```

I don't think we can change a tag msg once pushed, do we want to stay consistent and do 
```suggestion
      run: git tag -a v${{steps.package.outputs.version}} -m "releasing version v${{steps.package.outputs.version}}"
```
thoughts?

---

yeah I agree